lib/LedgerSMB/Scripts/recon.pm

```diff
@@ -274,10 +275,13 @@ sub _display_report {
 
     $recon->{zero_string} = LedgerSMB::PGNumber->from_input(0)->to_output(money => 1);
 
-    $recon->{statement_gl_calc} = $neg_factor *
-                                    ($recon->{their_total}
+    $recon->{statement_gl_calc} = $is_positive
+                                  ?  ($recon->{their_total}
```

Comment:
I've rechecked for what its worth.

I did NOT run a table through this passage, it was the PNL passage above.